src/pages/en/community/guidelines.mdx

```diff
@@ -0,0 +1,127 @@
+
+Authoritative policies can be found in our [RFCs](/about/faq#whats-an-rfc-how-do-they-work), along with our [Code of Conduct](/community/code-of-conduct) and [rules](/community/rules).
+If a guideline is in conflict with one of these, that document takes precedence over the guidelines.
+
+# Moderation
+
+<Message>
+	# Reporting a staff member
+
+	If you feel the need to report a member of staff, the right person to contact will depend on their roles:
+	- Developer: Contact ModMail or a Moderator directly
+	- Moderator: Contact a Manager or ModMail
+	- Manager: Contact a member of the Administrative Board or another Manager
+	- Administrative Board: Contact a Manager or another member of the Administrative Board
+</Message>
+
+<Message>
+	# Who can see a ModMail thread?
+
+	ModMail threads are, in most cases, only seen by Moderators and Community Managers. However, we may choose to share a thread with the Administrative Board on request,
+	for oversight purposes.
+
+	Application threads will also be seen by the Administrative Board and Community Team, as they are a part of the application process.
+
+	We may also choose to share a stripped down version of the thread with the Community Team, with any personal information redacted, if required as part of an
+	action against a member of the staff.
+
+	If we would like to share a thread further, we will ask for your permission first. Please note that bad faith threads are exempt from this rule.
+</Message>
+
+<Message>
+	# Food Policy
+
+	To avoid stigmatizing food, conversations about food and drinks are allowed anywhere in our community space. We ask you to not request
+	food content warnings, but you are free to content warn your own content as you wish.
+
+	We recognize that this policy may not be ideal for everyone, but we believe it is the approach that is most inclusive of our community and inline with our values.
+</Message>
+
+<Message>
+	# Syncing bans from Discord to other platforms
+
+	We take a very firm approach to moderation on Discord, as it is a social space before anything else. However, we can take a more lax approach
+	when it comes to other platforms to not block users entirely from voicing their opinions on Quilt's development or get support.
+
+	As such, we will sync bans from Discord to other platforms under any of the following conditions:
+	- The activity of the banned user in another platform would make vulnerable users feel uncomfortable and unable to interact in that platform.
+	- The user has been banned for spamming or trolling.
+	- We do not want our projects to be associated with that user.
+</Message>
+
+# Community
+
+<Message>
+	# Who can use PluralKit?
+
+	PluralKit is reserved for accessibility purpose. We allow any good faith usage of that tool under that condition.
+
+	For example:
+	- Allowed: Using PluralKit to distinguish between members of a plural system, endogenic or traumagenic, or questioning.
```

Comment:
I feel like this needs some sort of wording change but im not quite sure how to do it

I dislike the (what i read as) implication of only 3 possible "states" of plurality and a requirement to know your system origin, but it feels important to explicit about endogenic systems at least somewhere